File: android_plugins/notification/src/main/java/com/ironsrc/notify/NotificationReceiver.java
package com.ironsrc.notify;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class NotificationReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        try {
            NotificationHandler notificationHandler = new NotificationHandler();
            notificationHandler.handle(context, intent);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }

}

File: android_plugins/notificationKt/src/main/java/com/ironsrc/notify/NotificationReceiver.kt
package com.ironsrc.notify

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class NotificationReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        val notificationHandler = NotificationHandler(context, intent)

        try {
            notificationHandler.handle(intent.action)
        } catch (e: Exception) {
            e.printStackTrace()
        }
    }

}
